yt_dlp/extractor/tvw.py

```diff
@@ -115,3 +123,43 @@ def _real_extract(self, url):
                 'is_live': ('eventStatus', {lambda x: x == 'live'}),
             }),
         }
+
+
+class TvwTvChannelsIE(InfoExtractor):
+    IE_NAME = 'Tvw:TvChannels'
```

Comment:
```suggestion
    IE_NAME = 'tvw:tvchannels'
```

and add `IE_NAME = 'tvw'` to `TVWIE` for consistency